fish_prompt.fish

```diff
@@ -53,7 +54,12 @@
 function __bobthefish_git_branch -S -d 'Get the current git branch (or commitish)'
   set -l ref (command git symbolic-ref HEAD ^/dev/null)
   if [ $status -gt 0 ]
-    set -l branch (command git show-ref --head -s --abbrev | head -n1 ^/dev/null)
+    set -l branch
+    if [ "$theme_display_git_tag" = 'yes' ]
+      set branch (command git describe --tags ^/dev/null)
```

Comment:
this probably needs an `--always` in there to keep it from falling over if you don't have a tag in your repo yet.